src/main/java/ca/uhn/fhir/jpa/starter/CustomAuthorizationInterceptor.java

```diff
@@ -104,8 +104,7 @@
 			String kid = oAuth2Helper.getJwtKeyId(token);
 			publicKey = ObjectUtils.isEmpty(publicKey) ? oAuth2Helper.getJwtPublicKey(kid, OAUTH_URL) : publicKey;
 			JWTVerifier verifier = oAuth2Helper.getJWTVerifier(jwt, publicKey);
-			jwt = verifier.verify(token);
-			if (oAuth2Helper.verifyClientId(jwt, OAUTH_CLIENT_ID)) {			  
```

Comment:
The indentation of the following lines needs to change now that the `if` has been removed.